website/content/docs/templates/legacy_json_templates/user-variables.mdx

```diff
@@ -1,18 +1,18 @@
 ---
 description: |
-  User variables allow your templates to be further configured with variables
-  from the command-line, environment variables, or files. This lets you
-  parameterize your templates so that you can keep secret tokens,
-  environment-specific data, and other types of information out of your
-  templates. This maximizes the portability and shareability of the template.
-page_title: User Variables - Templates
+The `variables` block contains parameterized values for your templates. Learn how to configure `variables` to improve a template's security, portability, and shareability.
```

Comment:
```suggestion
  The `variables` block contains parameterized values for your templates. Learn how to configure `variables` to improve a template's security, portability, and shareability.
```

@trujillo-adam the spacing here is breaking rendering as multiline data needs the next line to be indented in.